evaluations/centrifuge-gsrpc-v2_1_gmajor-encrypt.md

```diff
@@ -0,0 +1,62 @@
+# Evaluation
+
+- **Status:** In Progress
+- **Application Document:** [centrifuge-gsrpc-v2](https://github.com/w3f/Grants-Program/blob/master/applications/centrifuge-gsrpc-v2.md)
+- **Milestone:** 1
+- **Kusama Identity:** [5DJ5EADF5yXoV6pY86HaDXYmH249VKXSo5jmAmHP5TGFzEjA](https://kusama.subscan.io/account/5DJ5EADF5yXoV6pY86HaDXYmH249VKXSo5jmAmHP5TGFzEjA)
+- **Previously successfully merged evaluation:** N/A
+
+| Number | Deliverable     | Accepted                | Link                                                                                                                                    | Evaluation Notes                                                                |
+|--------|-----------------|-------------------------|-----------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------|
+| 1.     | License         | <ul><li>[x] </li> </ul> | [License](https://github.com/centrifuge/go-substrate-rpc-client/blob/events-parsing-v2-milestone-1/LICENSE)                             | Apache License v2                                                               |
+| 2.     | Documentation   | <ul><li>[ ] </li></ul>  | [Documentation](https://github.com/centrifuge/go-substrate-rpc-client/blob/events-parsing-v2-milestone-1/README.md)                     | basically no documentation                                                      |
+| 3.     | PR with changes | <ul><li>[ ] </li></ul>  | [centrifuge/go-substrate-rpc-client#327](https://github.com/centrifuge/go-substrate-rpc-client/pull/327)                                | Lack of metadata v14 call support                                               |
+| 4a.    | Tests           | <ul><li>[ ] </li></ul>  | [UnitTest](https://github.com/centrifuge/go-substrate-rpc-client/blob/aa507a89484f162b47e1a2e6b71827307183b013/events/registry_test.go) | The test coverage rate is currently at 66%, which has not reached full coverage |
+| 4b.    | Test execution  | <ul><li>[x] </li></ul>  | [TestExecution](https://github.com/centrifuge/go-substrate-rpc-client/blob/aa507a89484f162b47e1a2e6b71827307183b013/Makefile#L49)       | pass                                                                            |
+
+
+## General Notes
+
+There are basically not any docs found in pr.
```

Comment:
We can add more inline documentation, but since this is an intermediate state that show case how to parse any metadata into a registry adding high level docs is will not be useful until Milestone 2 is completed, which will most likely refactor a bit the current code.
In the next milestone we are planning on adding proper end user documentation + inline doc as well. Since functions might change in between milestones we thought to do this in the final version.

Reply:
Will add more inline documentation for now.